modules/client/test/unit/client-common/utils.test.ts

```diff
@@ -76,3 +121,40 @@ describe("client-common utils", () => {
     });
   });
 });
+
+describe("Compute failing proposals", () => {
+  it("Should return true because the proposal changes the min approvals and then updates the addresses", async () => {
```

Comment:
```suggestion
describe("Detect failing proposals", () => {
  it("isFailingProposal should return true because the proposal changes the min approvals and then updates the addresses", async () => {
```